internal/utils/templates/init_config.toml

```diff
@@ -119,6 +119,20 @@ template = "Your code is {{ `{{ .Code }}` }} ."
 # enabled = true
 # uri = "pg-functions://<database>/<schema>/<hook_name>"
 
+# Use a custom sms sender
+[auth.hook.send_sms]
+# enabled = true
+# We use host.docker.internal to connect to port 54321 on the host. See: https://docs.docker.com/desktop/networking/
+# uri = "http://kong:8000/functions/v1/sms_sender"
+# Use this secret in the email_sender edge function as well.
+# secrets = "v1,whsec_c3VwYWJhc2Vpc3Zlcnlhd2Vzb21lYW5kaWxpa2VpdA=="
```

Comment:
How is this secret generated? If it's only a static value, may be we can hardcode it in config.go. Otherwise we want to use `env(SOME_SECRET)` substitution.

To avoid accidents and confusion, we don't want to add secret to config.toml even if they are only used locally.